package.json

```diff
@@ -16,6 +16,7 @@
     "glob": "^7.1.6",
     "husky": "^4.3.0",
     "lerna": "^4.0.0",
+    "minimist": "1.2.6",
```

Comment:
do we need this in dev deps?

Reply:
well it's not used by the app directly, and only exists to satisfy yarn audit...so yes?